DESCRIPTION

```diff
@@ -15,7 +15,8 @@ Depends:
     purrr
 Imports:
     utils,
-    grDevices,
-    httr
```

Comment:
it still imports httr, right?

Reply:
We import all needed functions in the namespace of the package also `httr` is in dependency of the package.